.github/workflows/fabric_acctest.yml

```diff
@@ -180,6 +180,7 @@ jobs:
           TF_ACC: "1"
           TF_ACC_FABRIC_CONNECTIONS_TEST_DATA: ${{ secrets.TF_ACC_FABRIC_CONNECTIONS_TEST_DATA }}
           TF_ACC_FABRIC_DEDICATED_PORTS: ${{ secrets.TF_ACC_FABRIC_DEDICATED_PORTS }}
+          TF_ACC_FABRIC_MARKET_PLACE_SUBSCRIPTION_ID: ${{ secrets.TF_ACC_FABRIC_MARKET_PLACE_SUBSCRIPTION_ID }}
```

Comment:
Is there going to be a resource for Fabric marketplace subscriptions as well?  How does a customer create or update a Fabric marketplace subscription?

Reply:
No, we are not planning to add any resources for Marketplace subscriptions. You can create a new subscription through the Cloud Service Provider Marketplace. Currently, we support AWS Marketplace subscription generation.
https://staging-docs.equinix.com/en-us/Content/Interconnection/FCR/FCR-order-csp-marketplace.htm?Highlight=fabric%20marketplace